frontend/src/components/pages/connect/Connector.Details.tsx

```diff
@@ -475,3 +490,138 @@ const ConnectorDetails = observer((p: {
         )}
     </Grid>
 });
+
+type LogsTabState = {
+    messages: TopicMessage[];
+    error?: string;
+    isComplete: boolean;
+};
+
+
+const LogsTab = observer((p: {
+    clusterName: string,
+    connectClusterStore: ConnectClusterStore,
+    connector: ClusterConnectorInfo,
+}) => {
+    const { connector } = p;
+    const connectorName = connector.name;
+    const topicName = '__redpanda.connectors_logs';
+    const topic = api.topics?.first(x => x.topicName == topicName);
+
+    const createLogsTabState = () => {
+        const s = observable({
+            messages: [],
+            isComplete: false,
+        } as LogsTabState);
+        // Start search immediately
+        const searchPromise = executeMessageSearch(topicName, connectorName);
+        searchPromise.catch(x => s.error = String(x)).then(() => s.isComplete = true);
+        // TODO:
+        // this is really unclean, the search api should return an *instance* of a search, which should
+        // be observable (containing an observable array of results, and error if any) but refactoring
+        // this in the same PR would be too much
+        s.messages = api.messages;
+        return s;
+    };
+
+    const [state, setState] = useState(createLogsTabState);
+
+    const paginationParams = usePaginationParams(10);
+    const messageTableColumns: ColumnDef<TopicMessage>[] = [
+        {
+            header: 'Timestamp',
+            accessorKey: 'timestamp',
+            cell: ({ row: { original: { timestamp } } }) => <TimestampDisplay unixEpochMillisecond={timestamp} format="default" />,
+            size: 30
+        },
+        {
+            header: 'Value',
+            accessorKey: 'value',
+            cell: ({ row: { original } }) => <MessagePreview msg={original} previewFields={() => []} isCompactTopic={topic ? topic.cleanupPolicy.includes('compact') : false} />,
+            size: 700
```

Comment:
I think in order to expand this column, we need to set this to Infinity or Number.MAX_SAFE_INTEGER https://tanstack.com/table/v8/docs/guide/column-sizing